src/main/java/net/pms/network/UPNPHelper.java

```diff
@@ -477,12 +477,18 @@ public void run() {
 
 							boolean redundant = address.equals(lastAddress) && packetType == lastPacketType;
 
-							if (packetType == M_SEARCH) {
-								if (configuration.getIpFiltering().allowed(address)) {
-									String remoteAddr = address.getHostAddress();
-									int remotePort = receivePacket.getPort();
+							if (configuration.getIpFiltering().allowed(address) && getDevice(address) != null) {
```

Comment:
@valib this change breaks the functionality - `getDevice` means it will only work for devices we know about, but this is useful for discovering _unknown_ devices